src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -237,6 +238,8 @@ public function __construct(array $attributes = [])
 
         $this->initializeTraits();
 
+        $this->applyCustomizations();
```

Comment:
See my previous command, if you rename this method to `initializeHasCustomizations`, it will be called automatically by the previous `initializeTraits` call, and then you can remove this call.